clients/pkg/logentry/stages/metrics.go

```diff
@@ -201,7 +201,7 @@ func (m *metricStage) recordCounter(name string, counter *metric.Counters, label
 		counter.With(labels).Inc()
 	case metric.CounterAdd:
 		f, err := getFloat(v)
-		if err != nil || f < 0 {
```

Comment:
That's weird; wonder why this was set. This one is for a counter - can we do a negative add in a counter? I'd assume so.

Reply:
🤷